src/index.mjs

```diff
@@ -175,3 +222,45 @@ export function prefetch(url, isPriority, conn) {
       })
   );
 }
+
+/**
+* Prerender a given URL
+* @param {String} url - the URL to fetch
+* @param {Object} [conn] - navigator.connection (internal)
+* @return {Object} a Promise
+*/
+export function prerender(urls, conn){
+  checkConnection(conn = navigator.connection).catch(err => {
+    return Promise.reject(new Error('Cannot prerender, '+err.message));
```

Comment:
Nothing is catching this promise rejection (since it is inside a catch handler).

Reply:
Thanks for catching this Domenic, should be fixed in 54eea9b07a7ef25c3b6102a36f9b8f6da6ae3c8c